### DISCUSSION TITLE
verify process

### DISCUSSION THREAD

**Discussion by FreshAir55** [FreshAirCoin/SmartContract#3] [Q&A] [answered]
Hello @cryptoguy55 
How did you verify this contract on etherscan?

**Comment by stackcraft55** [comment, marked as answer]
Hello.
At first time, It is needed to get the etherscan api
And then install the hardhat-etherscan module on hardhat project
Please look this articles
https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan
If you need further details , Please ping me
Thank you
